### ISSUE TITLE
[Bug] Recaptcha plugin is only retrieving some of the recaptchas on the site

### ISSUE THREAD

**Issue by notsofunctional** [berstend/puppeteer-extra#489]
**Describe the bug**

Hello,

I'm using the plugin **puppeteer-extra-plugin-recaptcha**,  but the method `page.findRecaptchas()` its only finding 1 of 4 recaptchas on the site. This site uses reCAPTCHA v2. I also tried to wait for more time until the page fully reloads, and i also try to find recaptchas frame by frame.

Any feedback?

**Code Snippet**

---

**Versions**

"puppeteer": "^9.0.0",
"puppeteer-extra-plugin-recaptcha": "^3.3.10",
"puppeteer-extra-plugin-stealth": "^2.7.7"

**Comment by berstend** [comment]
@notsofunctional could you run the plugin with DEBUG output enabled (explained in the readme) and post the output here?

It would also help if you can mention the site or [create a demo](https://github.com/berstend/berstend.github.io/tree/main/static/recaptcha) with a similar configuration.

**Comment by notsofunctional** [comment]
@berstend Sure, this is the site](https://listanominal.ine.mx/scpln/) i'm trying to solve recaptchas (not all, but just one per request), and this is likely the code I'm using:

```js
puppeteer.launch({
  headless: true,
  args: [
    '--ignore-certificate-errors',
    '--no-sandbox',
    '--disable-setuid-sandbox',
    '--disable-accelerated-2d-canvas',
    '--disable-gpu'
  ]
}).then(async (browser) => {
  const page = await browser.newPage()
  await page.goto('URL', { waitUntil : 'networkidle0' })

  console.log(await page.findRecaptchas())

  await browser.close()
})
```

And this is the debug ouput:

```
$ DEBUG=puppeteer-extra,puppeteer-extra-plugin:* node test.js
  puppeteer-extra-plugin:base:stealth Initialized. +0ms
  puppeteer-extra plugin registered stealth +0ms
  puppeteer-extra-plugin:base:recaptcha Initialized. +0ms
  puppeteer-extra-plugin:recaptcha Initialized {
  visualFeedback: false,
  throwOnError: false,
  provider: { id: '2captcha', token: 'TOKEN' }
} +0ms
  puppeteer-extra plugin registered recaptcha +2ms
  puppeteer-extra dependencies missing Set(15) {
  'stealth/evasions/chrome.app',
  'stealth/evasions/chrome.csi',
  'stealth/evasions/chrome.loadTimes',
  'stealth/evasions/chrome.runtime',
  'stealth/evasions/iframe.contentWindow',
  'stealth/evasions/media.codecs',
  'stealth/evasions/navigator.hardwareConcurrency',
  'stealth/evasions/navigator.languages',
  'stealth/evasions/navigator.permissions',
  'stealth/evasions/navigator.plugins',
  'stealth/evasions/navigator.webdriver',
  'stealth/evasions/sourceurl',
  'stealth/evasions/user-agent-override',
  'stealth/evasions/webgl.vendor',
  'stealth/evasions/window.outerdimensions'
} +0ms
  puppeteer-extra-plugin:base:stealth/evasions/chrome.app Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/chrome.app +2ms
  puppeteer-extra-plugin:base:stealth/evasions/chrome.csi Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/chrome.csi +0ms
  puppeteer-extra-plugin:base:stealth/evasions/chrome.loadTimes Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/chrome.loadTimes +1ms
  puppeteer-extra-plugin:base:stealth/evasions/chrome.runtime Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/chrome.runtime +0ms
  puppeteer-extra-plugin:base:stealth/evasions/iframe.contentWindow Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/iframe.contentWindow +1ms
  puppeteer-extra-plugin:base:stealth/evasions/media.codecs Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/media.codecs +1ms
  puppeteer-extra-plugin:base:stealth/evasions/navigator.hardwareConcurrency Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/navigator.hardwareConcurrency +0ms
  puppeteer-extra-plugin:base:stealth/evasions/navigator.languages Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/navigator.languages +1ms
  puppeteer-extra-plugin:base:stealth/evasions/navigator.permissions Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/navigator.permissions +0ms
  puppeteer-extra-plugin:base:stealth/evasions/navigator.plugins Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/navigator.plugins +2ms
  puppeteer-extra-plugin:base:stealth/evasions/navigator.webdriver Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/navigator.webdriver +0ms
  puppeteer-extra-plugin:base:stealth/evasions/sourceurl Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/sourceurl +1ms
  puppeteer-extra-plugin:base:stealth/evasions/user-agent-override Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/user-agent-override +0ms
  puppeteer-extra dependencies missing Set(3) {
  'stealth/evasions/webgl.vendor',
  'stealth/evasions/window.outerdimensions',
  'user-preferences'
} +0ms
  puppeteer-extra-plugin:base:stealth/evasions/webgl.vendor Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/webgl.vendor +1ms
  puppeteer-extra-plugin:base:stealth/evasions/window.outerdimensions Initialized. +0ms
  puppeteer-extra plugin registered stealth/evasions/window.outerdimensions +0ms
  puppeteer-extra-plugin:base:user-preferences Initialized. +0ms
  puppeteer-extra plugin registered user-preferences +1ms
  puppeteer-extra dependencies missing Set(1) { 'user-data-dir' } +0ms
  puppeteer-extra-plugin:base:user-data-dir Initialized. +0ms
  puppeteer-extra-plugin:user-data-dir initialized {
  deleteTemporary: true,
  deleteExisting: false,
  files: [],
  folderPath: '/tmp',
  folderPrefix: 'puppeteer_dev_profile-'
} +0ms
  puppeteer-extra plugin registered user-data-dir +12ms
  puppeteer-extra ignoring dependency 'stealth/evasions/webgl.vendor', which has been required already. +0ms
  puppeteer-extra ignoring dependency 'stealth/evasions/window.outerdimensions', which has been required already. +0ms
  puppeteer-extra orderPlugins:before [
  'stealth',
  'recaptcha',
  'stealth/evasions/chrome.app',
  'stealth/evasions/chrome.csi',
  'stealth/evasions/chrome.loadTimes',
  'stealth/evasions/chrome.runtime',
  'stealth/evasions/iframe.contentWindow',
  'stealth/evasions/media.codecs',
  'stealth/evasions/navigator.hardwareConcurrency',
  'stealth/evasions/navigator.languages',
  'stealth/evasions/navigator.permissions',
  'stealth/evasions/navigator.plugins',
  'stealth/evasions/navigator.webdriver',
  'stealth/evasions/sourceurl',
  'stealth/evasions/user-agent-override',
  'stealth/evasions/webgl.vendor',
  'stealth/evasions/window.outerdimensions',
  'user-preferences',
  'user-data-dir'
] +0ms
  puppeteer-extra orderPlugins:after [
  'stealth',
  'recaptcha',
  'stealth/evasions/chrome.app',
  'stealth/evasions/chrome.csi',
  'stealth/evasions/chrome.loadTimes',
  'stealth/evasions/chrome.runtime',
  'stealth/evasions/media.codecs',
  'stealth/evasions/navigator.hardwareConcurrency',
  'stealth/evasions/navigator.languages',
  'stealth/evasions/navigator.permissions',
  'stealth/evasions/navigator.plugins',
  'stealth/evasions/navigator.webdriver',
  'stealth/evasions/sourceurl',
  'stealth/evasions/user-agent-override',
  'stealth/evasions/webgl.vendor',
  'stealth/evasions/window.outerdimensions',
  'stealth/evasions/iframe.contentWindow',
  'user-preferences',
  'user-data-dir'
] +1ms
  puppeteer-extra-plugin:user-preferences _userPrefsFromPlugins { intl: { accept_languages: 'en-US,en' } } +0ms
  puppeteer-extra-plugin:user-data-dir created custom dir /tmp/puppeteer_dev_profile-YUe2Ql +2ms
  puppeteer-extra-plugin:user-data-dir Wrote file /tmp/puppeteer_dev_profile-YUe2Ql/Default/Preferences +2ms
  puppeteer-extra-plugin:recaptcha onPageCreated about:blank +0ms
  puppeteer-extra-plugin:stealth/evasions/user-agent-override onPageCreated - Will set these user agent options {
  override: {
    userAgent: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4469.0 Safari/537.36',
    platform: 'Win32',
    userAgentMetadata: {
      brands: [Array],
      fullVersion: '91.0.4469.0',
      platform: 'Windows',
      platformVersion: '10.0',
      architecture: 'x86',
      model: '',
      mobile: false
    },
    acceptLanguage: 'en-US,en'
  },
  opts: { userAgent: null, locale: 'en-US,en', maskLinux: true }
} +0ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +0ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +3ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +3ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +1ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +2ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +1ms
boot: 3.785s
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.evaluate' } +3ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +0ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +2ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +2ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +1ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +0ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +1ms
  puppeteer-extra-plugin:recaptcha solveRecaptchas +0ms
  puppeteer-extra-plugin:recaptcha findRecaptchas +0ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +2ms
  puppeteer-extra-plugin:recaptcha hasRecaptchaScriptTag true +0ms
  puppeteer-extra-plugin:recaptcha waitForRecaptchaClient - start 2021-04-26T16:04:09.866Z +0ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +1ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +1ms
  puppeteer-extra-plugin:recaptcha waitForRecaptchaClient - end 2021-04-26T16:04:09.868Z +0ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +0ms
  puppeteer-extra-plugin:recaptcha hasHcaptchaScriptTag false +0ms
  puppeteer-extra-plugin:recaptcha _generateContentScript recaptcha findRecaptchas undefined +0ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.evaluate' } +1ms
  puppeteer-extra-plugin:recaptcha _generateContentScript hcaptcha findRecaptchas undefined +0ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.evaluate' } +4ms
  puppeteer-extra-plugin:recaptcha findRecaptchas {
  captchas: [
    {
      sitekey: '6LdAe1sUAAAAACrdhVFHK5KmZ5TA8ZJ0iWQ6i64b',
      _vendor: 'recaptcha',
      id: 'sx916lj74upc',
      s: null,
      widgetId: 3,
      display: [Object],
      url: 'https://listanominal.ine.mx/scpln/',
      hasResponseElement: true
    }
  ],
  error: null
} +0ms
  puppeteer-extra-plugin:recaptcha getRecaptchaSolutions +0ms
  puppeteer-extra-plugin:recaptcha:2captcha Requesting solution.. {
  _vendor: 'recaptcha',
  provider: '2captcha',
  id: 'sx916lj74upc',
  requestAt: 2021-04-26T16:04:09.874Z
} +0ms
        puppeteer-extra-plugin:recaptcha:2captcha Got response {
  err: null,
  result: {
    id: '66736094841',
    text: '03AGdBq26V5KA4qA8-_UsoCMFkjJI5SDM8gerX1xXiKniIESlFDss_e8G6siC0_wY7KeLxYRLYflp0kEiV2woLL1SFRdH0vHW_6yEWGMYry2hpApSQQKTZAftuNU_HU02WzAQb53HOb1613BQO_YDB2Fxz4eFasbf8xA85EbQDW3yfKdMhg3B7Wd3Jn50z83VRPXwUZVP2jxwXPFY2NZx5N4WNHvDLi-qdZfPNAlmeSdYlOypE534NmKI19kUcgO-eIl0PtyGZpofHyVa6mhiwxm4Gb1eXlQLkRC2F8fbxmq1U79BJ5srJXr-IubS0Gw154Eptu_hMfFGaX4_4UOkwHBIALyrY4Ot-Cr5UtMZ9i4TYJLhPv3RP3grEAvYWMTCzw3TPTg8b91qo5UacvQbMDBH9U_xC5M1qsNF6t0CiKNy2NBPqQMNyKlJAuwaH1ZQX1KpVWJjeBBnP9h3G0WQFQ6zv84Ezu_pJwP1VuEOd8Mc9nDJxuOphvclkQBzu-ckJJMLgxtBxBqIszLUI4ZWlNjNFlknhpgoonbntGE76XYPv9nnCc4rSoMw'
  },
  invalid: [Function: bound ]
} +17s
  puppeteer-extra-plugin:recaptcha getRecaptchaSolutions {
  solutions: [
    {
      _vendor: 'recaptcha',
      provider: '2captcha',
      id: 'sx916lj74upc',
      requestAt: 2021-04-26T16:04:09.874Z,
      providerCaptchaId: '66736094841',
      text: '03AGdBq26V5KA4qA8-_UsoCMFkjJI5SDM8gerX1xXiKniIESlFDss_e8G6siC0_wY7KeLxYRLYflp0kEiV2woLL1SFRdH0vHW_6yEWGMYry2hpApSQQKTZAftuNU_HU02WzAQb53HOb1613BQO_YDB2Fxz4eFasbf8xA85EbQDW3yfKdMhg3B7Wd3Jn50z83VRPXwUZVP2jxwXPFY2NZx5N4WNHvDLi-qdZfPNAlmeSdYlOypE534NmKI19kUcgO-eIl0PtyGZpofHyVa6mhiwxm4Gb1eXlQLkRC2F8fbxmq1U79BJ5srJXr-IubS0Gw154Eptu_hMfFGaX4_4UOkwHBIALyrY4Ot-Cr5UtMZ9i4TYJLhPv3RP3grEAvYWMTCzw3TPTg8b91qo5UacvQbMDBH9U_xC5M1qsNF6t0CiKNy2NBPqQMNyKlJAuwaH1ZQX1KpVWJjeBBnP9h3G0WQFQ6zv84Ezu_pJwP1VuEOd8Mc9nDJxuOphvclkQBzu-ckJJMLgxtBxBqIszLUI4ZWlNjNFlknhpgoonbntGE76XYPv9nnCc4rSoMw',
      responseAt: 2021-04-26T16:04:26.723Z,
      hasSolution: true,
      duration: 16.849
    }
  ],
  error: undefined
} +0ms
  puppeteer-extra-plugin:recaptcha enterRecaptchaSolutions {
  solutions: [
    {
      _vendor: 'recaptcha',
      provider: '2captcha',
      id: 'sx916lj74upc',
      requestAt: 2021-04-26T16:04:09.874Z,
      providerCaptchaId: '66736094841',
      text: '03AGdBq26V5KA4qA8-_UsoCMFkjJI5SDM8gerX1xXiKniIESlFDss_e8G6siC0_wY7KeLxYRLYflp0kEiV2woLL1SFRdH0vHW_6yEWGMYry2hpApSQQKTZAftuNU_HU02WzAQb53HOb1613BQO_YDB2Fxz4eFasbf8xA85EbQDW3yfKdMhg3B7Wd3Jn50z83VRPXwUZVP2jxwXPFY2NZx5N4WNHvDLi-qdZfPNAlmeSdYlOypE534NmKI19kUcgO-eIl0PtyGZpofHyVa6mhiwxm4Gb1eXlQLkRC2F8fbxmq1U79BJ5srJXr-IubS0Gw154Eptu_hMfFGaX4_4UOkwHBIALyrY4Ot-Cr5UtMZ9i4TYJLhPv3RP3grEAvYWMTCzw3TPTg8b91qo5UacvQbMDBH9U_xC5M1qsNF6t0CiKNy2NBPqQMNyKlJAuwaH1ZQX1KpVWJjeBBnP9h3G0WQFQ6zv84Ezu_pJwP1VuEOd8Mc9nDJxuOphvclkQBzu-ckJJMLgxtBxBqIszLUI4ZWlNjNFlknhpgoonbntGE76XYPv9nnCc4rSoMw',
      responseAt: 2021-04-26T16:04:26.723Z,
      hasSolution: true,
      duration: 16.849
    }
  ]
} +0ms
  puppeteer-extra-plugin:recaptcha _generateContentScript recaptcha enterRecaptchaSolutions {
  solutions: [
    {
      _vendor: 'recaptcha',
      provider: '2captcha',
      id: 'sx916lj74upc',
      requestAt: 2021-04-26T16:04:09.874Z,
      providerCaptchaId: '66736094841',
      text: '03AGdBq26V5KA4qA8-_UsoCMFkjJI5SDM8gerX1xXiKniIESlFDss_e8G6siC0_wY7KeLxYRLYflp0kEiV2woLL1SFRdH0vHW_6yEWGMYry2hpApSQQKTZAftuNU_HU02WzAQb53HOb1613BQO_YDB2Fxz4eFasbf8xA85EbQDW3yfKdMhg3B7Wd3Jn50z83VRPXwUZVP2jxwXPFY2NZx5N4WNHvDLi-qdZfPNAlmeSdYlOypE534NmKI19kUcgO-eIl0PtyGZpofHyVa6mhiwxm4Gb1eXlQLkRC2F8fbxmq1U79BJ5srJXr-IubS0Gw154Eptu_hMfFGaX4_4UOkwHBIALyrY4Ot-Cr5UtMZ9i4TYJLhPv3RP3grEAvYWMTCzw3TPTg8b91qo5UacvQbMDBH9U_xC5M1qsNF6t0CiKNy2NBPqQMNyKlJAuwaH1ZQX1KpVWJjeBBnP9h3G0WQFQ6zv84Ezu_pJwP1VuEOd8Mc9nDJxuOphvclkQBzu-ckJJMLgxtBxBqIszLUI4ZWlNjNFlknhpgoonbntGE76XYPv9nnCc4rSoMw',
      responseAt: 2021-04-26T16:04:26.723Z,
      hasSolution: true,
      duration: 16.849
    }
  ]
} +0ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.evaluate' } +17s
  puppeteer-extra-plugin:recaptcha enterRecaptchaSolutions {
  solved: [
    {
      _vendor: 'recaptcha',
      id: 'sx916lj74upc',
      responseElement: true,
      responseCallback: false,
      isSolved: true,
      solvedAt: {}
    }
  ],
  error: undefined
} +0ms
  puppeteer-extra-plugin:recaptcha solveRecaptchas {
  captchas: [
    {
      sitekey: '6LdAe1sUAAAAACrdhVFHK5KmZ5TA8ZJ0iWQ6i64b',
      _vendor: 'recaptcha',
      id: 'sx916lj74upc',
      s: null,
      widgetId: 3,
      display: [Object],
      url: 'https://listanominal.ine.mx/scpln/',
      hasResponseElement: true
    }
  ],
  solutions: [
    {
      _vendor: 'recaptcha',
      provider: '2captcha',
      id: 'sx916lj74upc',
      requestAt: 2021-04-26T16:04:09.874Z,
      providerCaptchaId: '66736094841',
      text: '03AGdBq26V5KA4qA8-_UsoCMFkjJI5SDM8gerX1xXiKniIESlFDss_e8G6siC0_wY7KeLxYRLYflp0kEiV2woLL1SFRdH0vHW_6yEWGMYry2hpApSQQKTZAftuNU_HU02WzAQb53HOb1613BQO_YDB2Fxz4eFasbf8xA85EbQDW3yfKdMhg3B7Wd3Jn50z83VRPXwUZVP2jxwXPFY2NZx5N4WNHvDLi-qdZfPNAlmeSdYlOypE534NmKI19kUcgO-eIl0PtyGZpofHyVa6mhiwxm4Gb1eXlQLkRC2F8fbxmq1U79BJ5srJXr-IubS0Gw154Eptu_hMfFGaX4_4UOkwHBIALyrY4Ot-Cr5UtMZ9i4TYJLhPv3RP3grEAvYWMTCzw3TPTg8b91qo5UacvQbMDBH9U_xC5M1qsNF6t0CiKNy2NBPqQMNyKlJAuwaH1ZQX1KpVWJjeBBnP9h3G0WQFQ6zv84Ezu_pJwP1VuEOd8Mc9nDJxuOphvclkQBzu-ckJJMLgxtBxBqIszLUI4ZWlNjNFlknhpgoonbntGE76XYPv9nnCc4rSoMw',
      responseAt: 2021-04-26T16:04:26.723Z,
      hasSolution: true,
      duration: 16.849
    }
  ],
  solved: [
    {
      _vendor: 'recaptcha',
      id: 'sx916lj74upc',
      responseElement: true,
      responseCallback: false,
      isSolved: true,
      solvedAt: {}
    }
  ],
  error: undefined
} +0ms
recaptcha: 16.870s
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +10ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +31ms
  puppeteer-extra-plugin:stealth/evasions/sourceurl Stripping sourceURL { method: 'Runtime.callFunctionOn' } +19ms
```

**Comment by saimfd** [comment]
Having the same issues since yesterday, it was working fine before. Now its not even detecting the captcha. I also added wait before it but no luck

**Comment by berstend** [comment]
@saimfd which site? feel free to DM me on discord

**Comment by saimfd** [comment]
> @saimfd which site? feel free to DM me on discord

Sure what's your discord?

**Comment by berstend** [comment]
> Sure what's your discord?

https://extra.community 😄 

@notsofunctional I can confirm that not all recaptchas are extracted on that page, looking into this.

**Comment by notsofunctional** [comment]
I've contacted you on Discord.

Btw, at the end i managed the recaptchas by my hand, but i expect this issue to be resolved ;)

**Comment by berstend** [comment]
(the issue from @notsofunctional is resolved - the site in question switched to recaptcha.net which requires the latest recaptcha plugin to be used)

I found a bug that can result in not all recaptchas being detected when multiple are present and a couple of other things to optimize, new version with fixes will come soon.

**Comment by berstend** [comment]
@notsofunctional apologies for the slight delay, #496 has just landed and released as `puppeteer-extra-plugin-recaptcha@3.4.0`
